src/packages/core/extension-registry/conditions/switch.condition.ts

```diff
@@ -0,0 +1,54 @@
+import { UmbBaseController, UmbControllerHost } from '@umbraco-cms/backoffice/controller-api';
+import {
+	ManifestCondition,
+	UmbConditionConfigBase,
+	UmbExtensionCondition,
+} from '@umbraco-cms/backoffice/extension-api';
+
+export class UmbSwitchCondition extends UmbBaseController implements UmbExtensionCondition {
+	#timer?: any;
```

Comment:
Disable any for this line, since we dont have good types for setTimeout
```suggestion
	// eslint-disable-next-line @typescript-eslint/no-explicit-any
	#timer?: any;
```